src/openzaak/conf/includes/api.py

```diff
@@ -1,12 +1,20 @@
 # SPDX-License-Identifier: EUPL-1.2
 # Copyright (C) 2019 - 2022 Dimpact
-from vng_api_common.conf.api import *  # noqa - imports white-listed
+from vng_api_common.conf.api import (  # noqa: F401
+    BASE_REST_FRAMEWORK,
+    BASE_SWAGGER_SETTINGS,
+    COMMON_SPEC,
```

Comment:
Do we use this setting in OZ?

Reply:
I checked it, it's used by `use_external_components` command in vng_api_common, but I don't see that being used in OZ, so I'll remove this setting